docs/user/setup.rst

```diff
@@ -3,49 +3,65 @@
 Environment setup
 =================
 
-This is what you will be doing, in a nutshell:
+Scala Native has the following system requirements:
 
-* installation of sbt
-* installation of LLVM and Clang
-* installation of Boehm GC
+* Java 8 or later
+* sbt 0.13.15
+* LLVM
+* Boehm GC
 
 Installing sbt
 --------------
 
 Please refer to `this link <http://www.scala-sbt.org/release/docs/Setup.html>`_
-for instructions for your OS.
+for instructions for your operating system.
 
-Please note that you'll need Java 8 or more recent to use the Scala Native
-toolchain.
+Installing LLVM and Boehm GC
+----------------------------
 
-Installing LLVM, Clang and Boehm GC
------------------------------------
+LLVM and Boehm GC are Scala Native's only external dependencies. Here are
```

Comment:
I've also worked on updating this section to reflect that the Scala Native runtime and JDK implementation now depends on both Boehm GC and the RE2 regular expression engine.

https://github.com/scala-native/scala-native/pull/706/files#diff-a2d0368ffcd2931da4849e9d4288854e

Great work, BTW 👍